server/api/crud/projects.py

```diff
@@ -396,10 +399,26 @@ async def _calculate_pipelines_counters(
             return project_to_running_pipelines_count
 
         try:
-            _, _, pipelines = await fastapi.concurrency.run_in_threadpool(
-                server.api.db.session.run_function_with_new_db_session,
-                self._list_pipelines,
-            )
+            should_continue_next_page = True
+            next_page_token = ""
+            while should_continue_next_page:
+                (
+                    _,
+                    next_page_token,
+                    pipelines,
+                ) = await fastapi.concurrency.run_in_threadpool(
+                    server.api.db.session.run_function_with_new_db_session,
+                    self._list_pipelines,
+                    page_token=next_page_token,
+                )
+                should_continue_next_page = bool(next_page_token)
+                for pipeline in pipelines:
+                    if (
+                        pipeline["status"]
+                        not in mlrun.run.RunStatuses.stable_statuses()
+                    ):
+                        project_to_running_pipelines_count[pipeline["project"]] += 1
+
```

Comment:
Could be simplified a bit
```suggestion
            next_page_token = ""
            while True:
                (
                    _,
                    next_page_token,
                    pipelines,
                ) = await fastapi.concurrency.run_in_threadpool(
                    server.api.db.session.run_function_with_new_db_session,
                    self._list_pipelines,
                    page_token=next_page_token,
                )
                for pipeline in pipelines:
                    if (
                        pipeline["status"]
                        not in mlrun.run.RunStatuses.stable_statuses()
                    ):
                        project_to_running_pipelines_count[pipeline["project"]] += 1
               if not next_page_token:
                   break     

```